src/thyra/ThyraProvider.ts

```diff
@@ -20,13 +22,15 @@ import { IAccountDetails } from '../account';
 /**
  * The Thyra accounts url
  */
-export const THYRA_ACCOUNTS_URL =
-  'https://my.massa/thyra/plugin/massalabs/wallet/rest/wallet';
+export const THYRA_BASE_URL =
+  'https://my.massa/thyra/plugin/massalabs/wallet/rest/wallet/';
+
+export const THYRA_URL = 'http://my.massa';
```

Comment:
why not using https ?

Reply:
If it is 'https', the script cannot connect to thyra:
"""
Thyra nodes retrieval error Axios Error: Error: unable to verify the first certificate
[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Axios Error: Error: unable to verify the first certificate".] {
   code: 'ERR_UNHANDLED_REJECTION'
}
"""
I have to check if it works on another computer and I forgot to set it back to https